xapian-letor/tests/api_letor.cc

```diff
@@ -1414,3 +1414,80 @@ DEFINE_TESTCASE(createfeaturevector_tfdoclencolllfcolllen, generated)
 
     return true;
 }
+namespace Xapian {
+
+class XAPIAN_VISIBILITY_DEFAULT CustomFeature : public Feature {
```

Comment:
This class is only used within this one file in the testsuite, so it doesn't need to be in namespace `Xapian` (and indeed it's better really not to put it there).

It also doesn't need visibility annotations (those are there to allow us to restrict the symbols which are exported from the libxapianletor library to just those needed for the public API).

Reply:
fixed in a fixup commit.